src/views/Home.vue

```diff
@@ -107,6 +125,11 @@
       },
 
       methods: {
+          goToResearch() {
+            if(this.recherche !== ""){
+              this.$router.push({ path: 'recherche', query: { q: this.recherche }})
```

Comment:
Super l'utilisation de `query` :+1: 

Attention, il faut toujours privilégier les routes nommées plutôt que d'utiliser des chemins en dur.

```suggestion
              this.$router.push({ name: 'search', query: { q: this.recherche }})
```